manager/testdata/new-blueprints/scenario4/plotter.yaml

```diff
@@ -56,6 +56,44 @@ spec:
         bucket: srcbucket
         object: inventory.parq
     dataformat: parquet
+  - assetId: "m4d-notebook-sample/paysim-step2" # the sink of step 2, consumed by step 1
+    cluster: thegreendragon
+    connection: 
+      arrow-flight:
+        endpoint:  app1-ns1-arrow-transform.m4d-blueprints
```

Comment:
@revit13 Do we know what the endpoint will be prior to deploying the blueprints?  The endpoint will differ per cluster I assume.  I don't see an indication of that in the example.

Reply:
We know the endpoint prior to deploying the blueprint. It is based on 1) cluster 2) release name 3) module specifics defined in FybrikModule.